fili-core/src/main/java/com/yahoo/bard/webservice/data/config/metric/MetricLoader.java

```diff
@@ -18,7 +20,11 @@
      * @deprecated in favor of loadMetricDictionary(MetricDictionary, DimensionDictionary)
      */
     @Deprecated
-    void loadMetricDictionary(MetricDictionary metricDictionary);
+    default void loadMetricDictionary(MetricDictionary metricDictionary) {
+        String message = "The method loadMetricDictionary(MetricDictionary, DimensionDictionary) should be used " +
+                "instead";
+        LoggerFactory.getLogger(MetricLoader.class).warn(message);
+    }
```

Comment:
Actually I don't think I want to do that, because I don't want people implementing this interface to use this Logger and this Logger can not be private.